src/components/todoItem/TodoItem.tsx

```diff
@@ -0,0 +1,62 @@
+import cn from 'classnames';
+
+interface Props {
+  title: string;
+  id: number;
+  completed: boolean;
+  onDelete: (id: number) => void;
+  onIsComplitedUpdate: (
+    id: number,
+    complitedCurrVal: boolean,
+  ) => void;
+  loading: boolean;
+  loadingID: number;
+}
+
+export const TodoItem: React.FC<Props> = ({
+  title,
+  id,
+  completed,
+  onDelete,
+  onIsComplitedUpdate,
+  loading,
+  loadingID,
+}) => {
+  return (
```

Comment:
It's better to use the 'React.memo' function to wrap the 'TodoItem' component to optimize rendering performance.